packages/listview/src/components/NavigationButtonColumn/styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+// eslint-disable-next-line import/prefer-default-export
+export const StyledButton = styled.button`
```

Comment:
use attachThemeAttr function to avoid crash when not used inside the Application component